src/processes.c

```diff
@@ -592,6 +602,18 @@ static int ps_config (oconfig_item_t *ci)
 			ps_list_register (c->values[0].value.string,
 					c->values[1].value.string);
 		}
+		else if (strcasecmp (c->key, "CollectFileDescriptor") == 0)
+		{
+
+			if ((c->values_num != 1)
```

Comment:
Can you please use `cf_util_get_boolean()` here? It will do the validation and logging for you.